containers/ddev-php-base/generic-files/usr/local/bin/install_php_extensions.sh

```diff
@@ -24,4 +24,5 @@ pkgs=$(yq ".${PHP_VERSION//.}.${ARCH} | join(\" \")" /etc/php-packages.yaml | aw
 # Echo the packages to be installed for logging
 echo "Installing packages for PHP ${PHP_VERSION} on ${ARCH}: $pkgs"
 
-apt-get -qq install --no-install-recommends --no-install-suggests -y $pkgs || exit $?
+# TODO: I'm not convinced that the apt-get update should be here or outside the script
+(apt-get -qq update || true) && DEBIAN_FRONTEND=noninteractive apt-get -qq install --no-install-recommends --no-install-suggests -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y $pkgs || exit $?
```

Comment:
I tried this without the Internet, and the Docker build failed only after 4m30s (something changed in Docker again, because it used to fail faster, I see the same long timeout with v1.23.5).

Let's update just the `deb.sury.org` repo here to make it faster (the same thing is done in [mariadb-client-install.sh](https://github.com/ddev/ddev/blob/da658c0814ddc6a673f7a388e2299f8b081e9adc/containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/mariadb-client-install.sh#L25)):

```suggestion
apt-get -qq update -o Dir::Etc::sourcelist="sources.list.d/php.list" -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0" || exit $?
DEBIAN_FRONTEND=noninteractive apt-get -qq install --no-install-recommends --no-install-suggests -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y $pkgs || exit $?
```

Reply:
You are the greatest. I didn't even know it was possible to do that. In the oldest database images we actually just remove the list from sources.list.d to make it invisible because the sources are no longer available or don't work anyway. This is a very nice technique.